google/resource_container_cluster_test.go

```diff
@@ -1770,3 +1796,44 @@ resource "google_container_cluster" "with_pod_security_policy" {
 	}
 }`, clusterName, enabled)
 }
+
+func testAccContainerCluster_withPrivateCluster(clusterName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "container_network" {
+	name = "container-net-%s"
+	auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "container_subnetwork" {
+	name          = "${google_compute_network.container_network.name}"
+	network       = "${google_compute_network.container_network.name}"
+	ip_cidr_range = "10.0.36.0/24"
+	region        = "us-central1"
+
```

Comment:
I ran the tests, and it looks like `private_ip_google_access` is getting enabled automatically on the subnetwork when the cluster is created in it. Mind just setting that to true from the beginning?

Reply:
Added